src/views/Gamefi/Gamefi.tsx

```diff
@@ -149,7 +149,7 @@ const Gamefi: React.FC = () => {
     const render = (type) => {
       switch (type) {
         case 'RENDER_ENDED':
-          return (
+          return stakedMemoized.inactiveFarms.length !== 0 && stakedMemoized.inactivePools.length !== 0 ? (
             <>
```

Comment:
iwasan ang nested conditions using ternary operator kasi nagiging confusing

try mo si '&&' approach

Reply:
Di man yan kuys nested hahah. Dual lang
First: check if both meron para magrender isa lang. 

Diba meron ata ditong linting maga show naamn if nested or maga throw error

Reply:
Mag render isa lang na error message ( not available component ) 


Or alisin ko nalang as is na pag both wala, dalawa rin error message ?